Library/Subscription.cs

```diff
@@ -422,33 +422,28 @@ public void Preview()
         /// Preview the changes associated with the current subscription
         /// </summary>
         /// <param name="timeframe">ChangeTimeframe.Now (default) or at Renewal</param>
-        public virtual Subscription PreviewChange(ChangeTimeframe timeframe)
+        /// <param name="preds">specify the fields that are changed and should be sent to the server, 
+        /// null sends all fields for backwards compatibility.</param>
+        public virtual Subscription PreviewChange(ChangeTimeframe timeframe, params Expression<Func<Subscription, object>>[] preds)

```

Comment:
Do you think it makes more sense naming the parameter `predicates` instead of `preds`? Longer name but clearer for the person implementing.